src/core/dbus.c

```diff
@@ -984,6 +988,19 @@ int bus_init_private(Manager *m) {
         return 0;
 }
 
+static int bus_track_stash(sd_bus_track *t, char ***l) {
+        for (const char *n = sd_bus_track_first(t); n; n = sd_bus_track_next(t)) {
+                int c, j;
+
+                c = sd_bus_track_count_name(t, n);
+                for (j = 0; j < c; j++)
+                        if (strv_extend(l, n) < 0)
+                                log_oom();
```

Comment:
Hmm, so we use repeated strings in the strv to specify the count. This seems iffy. Why not use a hashmap name→count instead?

Edit: Oh, OK, this is because that's how `deserialized_subscribed` is used already. I guess that's OK, if the repeat counts are not too high.